src/components/Header/components/NavButtonsRightCorner/index.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../../../vars';
+
+#right-menu-triangle {
+  position: absolute;
+  right: 1.41rem;
+  top: -0.55rem;
+  background-color: white;
+  height: 1.1rem;
+  width: 1.1rem;
```

Comment:
Yeah lol. 1 rem is equivalent to 16px. I used rem because it's generally used for spacing and sizing.

Reply:
So I did some research and this is a tough decision. Generally speaking, it's correct and best to use rem instead of px to allow the user to choose their own font size and have the website scale according to that number. However, many of 360's CSS files probably use px instead of rem which would defeat the purpose. So I'm not sure if I should leave the rem to implement best practice or change it to make it similar to past CSS files.

Reply:
Lol, I read the same exact website haha. I'll definitely test out changing the font-size of the document right now. Hopefully, nothing changes haha!

Reply:
Oh gosh, the same happened to me (I changed the font-size to 40px haha)! Some things still worked perfectly fine on pages such as Events but broke with Involvements and other pages. While the text still appears the same size, the spacing around it completely changes. So I guess I'll keep the rems for now and yes, this is definitely a future project lol